app/controllers/devise/sessions_controller.rb

```diff
@@ -77,7 +77,7 @@ def respond_to_on_destroy
     # support returning empty response on GET request
     respond_to do |format|
       format.all { head :no_content }
-      format.any(*navigational_formats) { redirect_to after_sign_out_path_for(resource_name) }
+      format.any(*navigational_formats) { redirect_to after_sign_out_path_for(resource_name), status: :see_other }
```

Comment:
Note: responders now has a configuration `redirect_status` that allows us to override this across the board.

This status is also going to be necessary on the registrations controller destroy action:
https://github.com/heartcombo/devise/blob/3632ddf67473880ed9583201ce13f108c620514f/app/controllers/devise/registrations_controller.rb#L70